service/controller/v15/resource/bridgezone/resource.go

```diff
@@ -30,9 +31,9 @@ type Config struct {
 }
 
 // Resource is bridgezone resource making sure we have fallback delegation in
-// old DNS structure. This is only for the migration period. When we delete the
-// "intermediate" zone this resource becomes noop and we do not need it
-// anymore.
+// old DNS structure. TODO This is only for the migration period. TODO When we
+// delete the "intermediate" zone this resource becomes noop and we do not need
+// it anymore.
```

Comment:
Just added `TODO` here for the diff so we can talk. The comment states this resource is temporary and can be removed at some point. When has this point come? There is no TODO issue as we agreed on having them so we should add this now. Who can tell more about the time horizon here? /cc @kopiczko @TheoBrigitte